cogs/hostbot.py

```diff
@@ -180,6 +181,21 @@ async def init_server(self, ctx: commands.Context, *, yml_config: str):
                 )
                 logging.info(f"graveyard: creating {channel_name} for {key}")
 
+            if key == "confessionals":
+                overwrites[ctx.guild.default_role] = discord.PermissionOverwrite(
+                    read_messages=False, send_messages=None
+                )
+                overwrites[roles["dead"]["role"]] = discord.PermissionOverwrite(
+                    read_messages=True, send_messages=False
+                )
+                overwrites[roles["host"]["role"]] = discord.PermissionOverwrite(
+                    read_messages=True
+                )
+                overwrites[roles["spec"]["role"]] = discord.PermissionOverwrite(
+                    read_messages=True, send_messages=False
+                )
```

Comment:
It's probably slightly more correct to set `@everyone` `send_msssages` to `False` and only set the host role to `True` but I'd want to see this tested locally to make sure it works.

I also don't really care.